src/python/turicreate/toolkits/activity_classifier/_tf_model_architecture.py

```diff
@@ -112,46 +126,75 @@ def __init__(self, net_params, batch_size, num_features, num_classes, prediction
         # Initialize all variables
         init = _tf.compat.v1.global_variables_initializer()
         self.sess.run(init)
-
+        
         # Assign the initialised weights from MXNet to tensorflow 
         for key in net_params.keys():
             if key in weights.keys():
                 if key.startswith('conv'):
-                    self.sess.run(_tf.assign(_tf.get_default_graph().get_tensor_by_name(key+":0"), _np.transpose(net_params[key].asnumpy(), (2, 1, 0))))
+                    net_params[key] = _np.reshape(_np.transpose(net_params[key], (3, 1, 0, 2)), (prediction_window, self.num_features, _net_params['conv_h']))
```

Comment:
Could we factor our this conversion into a utility function that can be used by other toolkits?

Reply:
But this is 1D Convs and like OD/DC use 2D Convs. They would all be named conv0_weight/conv1_weight or just conv_weight!